apps/docs/src/app/pages/table/implementation/header/ng-doc.page.ts

```diff
@@ -0,0 +1,16 @@
+import { NgDocPage } from '@ng-doc/core';
+import TableImplementationCategory from 'apps/docs/src/app/categories/table/sub-categories/implementation/ng-doc.category';
```

Comment:
TODO: make these paths relative instead of absolute from the root of the project.

Perhaps add a barrel file in `apps/docs/src/app/categories` so that the import could be:
```
import { TableImplementationCategory } from '../../../../categories';
```

Reply:
This comment also applies to other places where this occurs.